test/Serilog.Sinks.MSSqlServer.Tests/DatabaseFixture.cs

```diff
@@ -42,7 +42,7 @@ public DatabaseFixture()
         
         public void Dispose()
         {
-            DeleteDatabase();
+            //DeleteDatabase();
```

Comment:
Sometimes (most of the time) I put it back... I think the right approach is to leave it in there and try not to commit it again :)